keras/src/models/sequential.py

```diff
@@ -226,12 +228,16 @@ def call(self, inputs, training=None, mask=None):
             # `outputs` are the outputs of `layer` applied to `inputs`. At the
             # end of each iteration `inputs` is set to `outputs` to prepare for
             # the next layer.
-            kwargs = {}
+            layer_kwargs = {}
             if layer._call_has_mask_arg:
-                kwargs["mask"] = mask
+                layer_kwargs["mask"] = mask
             if layer._call_has_training_arg and training is not None:
-                kwargs["training"] = training
-            outputs = layer(inputs, **kwargs)
+                layer_kwargs["training"] = training
+            for flag, value in kwargs.items():
```

Comment:
Rename `flag`.

Maybe it would be more readable if you looped on `_call_has_flag_arg` instead?

```python
layer_kwargs = {k: kwargs[k] for k in getattr(layer, "_call_has_flag_arg", {}) if k in kwargs}
```

Reply:
Done.